mora/service/orgunit.py

```diff
@@ -81,7 +81,10 @@ def get(self, objid):
                     ),
                 },
             )
-            if c.is_effect_relevant({'from': start, 'to': end})
+            if c.is_effect_relevant({'from': start, 'to': end}) and
+            effect.get('tilstande')
+                  .get('organisationenhedgyldighed')[0]
+                  .get('gyldighed') == 'Aktiv'
```

Comment:
I think we have loads of statements like this, and it's actually kind of horrible. Would be nice to do `c.has_state(effect, 'Aktiv')` instead…

Reply:
Yes, I agree. Let's do that at some point...